src/components/SharedComponents/PhotoCount.js

```diff
@@ -32,19 +38,23 @@ const PhotoCount = ( { count, size, shadow }: Props ): React.Node => {
           fillRule="nonzero"
           clipRule="evenodd"
           d="M4 5.818a4 4 0 00-4 4V20a4 4 0 004 4h10.182a4 4 0 004-4V9.818a4 4 0 00-4-4z"
-          fill="#fff"
-          filter="url(#drop-shadow)"
+          fill={colors.white}
         />
         <Path
           // eslint-disable-next-line
```

Comment:
Haven't let a comment here before. But if you disable eslint, please be specific about which rule.